presentation/health.py

```diff
@@ -0,0 +1,24 @@
+from typing import List
+
+from domain.model import HealthItem, MeshResults
+from domain.types import AgentID
+
+
+class Health:
```

Comment:
I suggest to make the extraction of test results for particular agents and metric be part of the `MeshResults` class. The interface should be something like:

```
def filter(from_agent: AgentID, to_agent: AgentID, metric: Metric) -> List[datetime, float]:
   ...
```

The "health" terminology is misplaced here. "Health" (in the context of network monitoring) is a judgement based on data. What the "Health" class stores are actual observations.

Reply:
After discussion, moving filtering functionality into MeshResults